### ISSUE TITLE
关于IoU&NMS

### ISSUE THREAD

**Issue by theDeep1nteresting** [bubbliiiing/yolox-pytorch#31]
hi up : )
请教两个问题：
1、关于IoU，想问一下：为什么这里IoULoss是 1 - iou ** 2而不是1 - iou 
        if self.loss_type == "iou":
            loss = 1 - iou ** 2
2、关于NMS，想问一下：utils_bbox.py文件下的有一大段注释内容，请问是做什么用的？
视频内做：Decode-特征点解码过程可视化的文件，和现在的不一样
